web/styles/settings.css

```diff
@@ -566,14 +566,19 @@ input[type="checkbox"] {
     }
 }
 
-.edit-attachment-buttons {
```

Comment:
dont think we can remove this
![uploaded](https://github.com/zulip/zulip/assets/40122794/db26fdce-798e-4a02-bbd1-2fa5b8118e08)

Reply:
Yes that was due to a merge conflict. I reverted the changes.